site/src/pages/index.tsx

```diff
@@ -0,0 +1,402 @@
+            from={
+              <OutboundLink href="https://en.wikipedia.org/wiki/Afterpay">
+                Afterpay
+              </OutboundLink>
+            }
+            bgColor1="rgba(255, 255, 255, 0.5)"
+            bgColor2="white"
+          >
+            Armeria has eased our adoption of gRPC at Afterpay. Serving gRPC,
+            documentation, health checks and metrics on a single port (with or
+            without TLS) gives us the functionality of a sidecar like Envoy in a
+            single process. Automated serving of API documentation, complete
+            with sample payloads, allows simple ad-hoc gRPC requests straight
+            from the browser. Distributed tracing is trivial to configure using
+            the built-in Zipkin decorators. Logging request bodies, performing
+            retries or circuit breaking are all built-in. The friendly devs
+            rapidly respond to issues and feature requests. Armeria is becoming
+            the default stack for gRPC at Afterpay.
+          </Blockquote>
+        </MarketingBlock>
+      </Marketing>
+      <Marketing>
+        <MarketingBlock>
+          <Title level={1}>
+            Fuel your request pipeline with{' '}
+            <Highlight>reusable components</Highlight>
+          </Title>
+          <Paragraph>
+            There are so many cross-cutting concerns you need to address when
+            building a microservice &mdash; metrics, distributed tracing, load
+            balancing, authentication, rate-limiting, circuit breakers,
+            automatic retries, etc.
+          </Paragraph>
+          <Paragraph>
+            Armeria solves your problems by providing various reusable
+            components called &lsquo;decorators&rsquo; in a clean, consistent,
+            flexible and user-friendly API. You can also write a custom
+            decorator if ours doesn&apos;t work for your use case.
+          </Paragraph>
+        </MarketingBlock>
+        <MarketingBlock>
+          <CodeBlock language="java">
+            {`
+            Server
+              .builder()
+              .service((ctx, req) -> ...);
+              .decorator(
+                MetricCollectingService.newDecorator(...))
+              .decorator(
+                BraveService.newDecorator(...))
+              .decorator(
+                AuthService.newDecorator(...))
+              .decorator(
+                ThrottlingService.newDecorator(...))
+              .build()
+              .start();
+            `}
+          </CodeBlock>
+        </MarketingBlock>
+      </Marketing>
+      <Marketing reverse>
+        <MarketingBlock>
+          <Title level={1}>
+            <Highlight>Integrate seamlessly</Highlight> with your favorite
+            frameworks &amp; languages
+          </Title>
+          <Paragraph>
+            Wanna try some cool technology while using Spring Boot or
+            Dropwizard? That&apos;s absolutely fine. Armeria integrates with
```

Comment:
Maybe integrates them without getting in the way.

Not sure about this "not like any other that claims so."

Reply:
I meant 'without enforcing a user to use certain features shipped with the framework'. 'integrates them without getting in the way' sounds good to me. Let me know if you find a better phrase.